### PR TITLE
January TGU - First Draft

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Happy birthday, Nush
image
## Why It's Good For The Game

Makes 515 work, among other things.
## Changelog
:cl:
/:cl:

### CODE REVIEW

**Comment by FernandoJ8** [comment]
Are you fucking with me, does git not track that name change

**Review by JohnFulpWillard — COMMENTED** [review]
check if bloodsucker guardian works please

File: fulp_modules/features/clothing/wintercoats/code/wintercoats.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	cold_protection = CHEST|GROIN|LEGS|ARMS
 	heat_protection = CHEST|GROIN|LEGS|ARMS
 	allowed = list(/obj/item/paper, /obj/item/soap, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
-	armor = list(MELEE = 10, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0)
+	armor_type = /datum/armor/wintercoat_miner
```
Comment (JohnFulpWillard) [inline fulp_modules/features/clothing/wintercoats/code/wintercoats.dm:15]: do regular wintercoats not have their own armor? kinda weird to give them miner armor

File: fulp_modules/features/antagonists/infiltrators/infil_objectives.dm [hunk 2]
```diff
@@ -144,7 +144,7 @@
 		/mob/living/basic/pet/dog/corgi/ian,
 		/mob/living/basic/pet/dog/corgi/puppy/ian,
 		/mob/living/basic/pet/dog/pug/mcgriff,
-		/mob/living/simple_animal/hostile/carp/lia,
+		//mob/living/basic/carp/pet/lia,
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/infiltrators/infil_objectives.dm:147]: was commenting this out intentional

File: fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_recipes.dm [hunk 3]
```diff
@@ -10,7 +10,7 @@
 		/obj/item/stack/sheet/iron = 1,
 	)
 	time = 15 SECONDS
-	category = CAT_PRIMAL
+	category = CAT_FURNITURE
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_recipes.dm:13]: Set to ``CAT_STRUCTURE`` please, that's what I did in my currently open PR and they aren't furniture.
I'll revert the change on my own PR

File: fulp_modules/features/species/beefmen/mob/beefman_organs.dm [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+/obj/item/organ/internal/brain/beefman/Insert(mob/living/carbon/C, special, drop_if_replaced, no_id_transfer)
+	var/creating_traumas = TRUE
+	for(var/datum/brain_trauma/trauma in traumas)
+		if(istype(trauma, /datum/brain_trauma/special/bluespace_prophet/phobetor))
+			creating_traumas = FALSE
+	if(creating_traumas)
+		if(C.dna.features["beef_trauma"])
+			C.gain_trauma(C.dna.features["beef_trauma"], TRAUMA_RESILIENCE_ABSOLUTE)
+		C.gain_trauma(/datum/brain_trauma/special/bluespace_prophet/phobetor, TRAUMA_RESILIENCE_ABSOLUTE)
+	. = ..()
```
Comment (JohnFulpWillard) [inline fulp_modules/features/species/beefmen/mob/beefman_organs.dm:14]:
```suggestion
	return ..()
```

File: fulp_modules/features/species/beefmen/mob/beefman_organs.dm [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/obj/item/organ/internal/brain/beefman
+	name = "minced-meat brain"
+	desc = "The inexplicably functional brain of a beefman. Some parts seem beyond any dream of repair though..."
+
+/obj/item/organ/internal/brain/beefman/Insert(mob/living/carbon/C, special, drop_if_replaced, no_id_transfer)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/species/beefmen/mob/beefman_organs.dm:5]: single letter var in MY fulpcode? it's more likely than you think!

**Comment by StoneMason** [comment]
bro i want goliath arm pls

File: fulp_modules/features/species/beefmen/mob/beefman_organs.dm (hunk 5 above)
Comment (remove32) [reply fulp_modules/features/species/beefmen/mob/beefman_organs.dm:5]: 😭

**Comment by SgtHunk** [comment]
i dont care its probably still january somewhere

**Pull request merged**